.github/workflows/ci.yml

```diff
@@ -180,7 +180,7 @@ jobs:
           AZURE_STORAGE_KEY: ${{ secrets.AZURE_STORAGE_KEY }}
           AZURE_BUCKET_NAME: ${{ secrets.AZURE_BUCKET_NAME }}
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_COMMUNITY }}
-          TELEMETRY_ENDPOINT: oss.edge.df.f5.com:443
+          TELEMETRY_ENDPOINT: ${{ github.ref_type == 'branch' && github.event_name == 'push' && github.ref == 'refs/heads/release' && 'oss-dev.edge.df.f5.com:443' || 'oss.edge.df.f5.com:443' }}
```

Comment:
`github.ref == 'refs/heads/release'` won't work because it has to start with release, but not equal release. You should be able to use the `startsWith()` function.